wordpress-mobile/gutenberg-mobile
# Revert changes to the jetpack reference that were caused by a git client

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/pull/3380#issuecomment-843338960

Reverts unneeded changes to the jetpack reference. 

To test:
Verify that the jetpack reference is now `a2869e608b6990b5e36d723ede4b7391c943348a` as it was previously. It was mistakenly updated when this PR was [merged](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3380).
PR submission checklist:
